hypervisor/arch/x86/e820.c

```diff
@@ -17,33 +17,34 @@
  * and hide HV memory from SOS_VM...
  */
 
-static uint32_t e820_entries_count;
-static struct e820_entry e820[E820_MAX_ENTRIES];
-static struct e820_mem_params e820_mem;
+static uint32_t hv_e820_entries_nr;
+static struct e820_entry hv_e820[E820_MAX_ENTRIES];
+static struct mem_range hv_mem_range;
```

Comment:
Add a comment too, like: "Describe the top/bottom/size of the physical memory the hypervisor manages */

Reply:
Sure